jobs/search-term-data-validation/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.4"
+
+services:
+  app:
+    build:
+      context: .
+    volumes:
+      - ./:/app
+      # Mount the local gcloud sdk configuration when developing. Note that this
+      # will modify the contents on the host.
+      - ${CLOUDSDK_CONFIG}/:/tmp/.config/gcloud
+    environment:s
```

Comment:
is the `:s` correct syntax? it seems to be breaking the yaml parsing in gh but i'm no yaml expert so it could be 🤷

Reply:
I more or less copied the structure of `kpi-forecasting`, which has it!